app/services/quantified_events/validate_creation_service.rb

```diff
@@ -26,7 +26,10 @@ def valid?
     delegate :customer, to: :subscription
 
     def operation_type
-      @operation_type ||= args.dig('properties', 'operation_type')&.to_sym
+      @operation_type ||= begin
+        type = args.dig('properties', 'operation_type')&.to_sym
+        type.nil? && billable_metric.unique_count_agg? ? :add : type
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.